cpp/ps0002/ps0002_soln.cpp

```diff
@@ -0,0 +1,46 @@
+#include <iostream>
+using namespace std;
+
+int main()
+{
+    /*
+    string Person;
+    int id;
+    string name;
+    int phone;
+    string gender;
+    */
+    unsigned short population = 40; // 0/0.75
+    unsigned short teammates = 10; // 0/0.75
+    unsigned short alternatives = 5; // 0/0.75
+
+    float pie = 5.5f; // 0/0.75
+    
+    char ch = 'h'; // 0.75/0.75
+  
+    unsigned int births= 14120000; // 0/0.75
+    
+    /*
+    float x=0;
+    cin >>x;
+    cout << "x="<<(x/7)<<endl;
+    */
+    long double div = 45 / 7; // 0.5/0.75
+  
+    float invoice = 5555.50; // 0/0.75
+    
+    /*
+    int age;
+    cin >> age;
+    int age_in_days=age*365;
+    int age_in_hours=age_in_days*24;
+    int age_in_scend=age_in_hours*60*60;
+    */
+    unsigned int age_in_seconds = 20 * 365 * 24 * 60 * 60; // 0.5/0.75
+
+    // total 1.75/6.75
```

Comment:
- [ ] all variables required in the document are declared
- [ ] variables are declared with the correct data type
- [x] using correct literals and notations for assigning initial values to the variables